Lib/os.py

```diff
@@ -490,8 +490,8 @@ def fwalk(top=".", topdown=True, onerror=None, *, follow_symlinks=False, dir_fd=
 
     def _fwalk(stack, isbytes, topdown, onerror, follow_symlinks):
         # Note: This uses O(depth of the directory tree) file descriptors: if
-        # necessary, it can be adapted to only require O(1) FDs, see issue
-        # #13734.
+        # necessary, it can be adapted to only require O(1) FDs.
+        # See https://github.com/python/cpython/issues/57943
```

Comment:
```suggestion
        # See https://github.com/python/cpython/issues/57943.
```

This is really me being nitpicky but I consider it a sentence and sentences end with a period :)

Reply:
> This is really me being nitpicky but I consider it a sentence and sentences end with a period :)

He says without a period 😄 

not to worry, I'll open fewer PRs in the evening and make sure I remember this one for next time. ;)

Reply:
(the smiley is the period!)